### PR TITLE
add shift tab command

### PR DESCRIPTION
**Pull request by alexboche**
Maybe I missed something, but it doesn't look like caster has a shift tab command so I added one called shift tab or shin tab.

### CODE REVIEW

File: castervoice/lib/ccr/core/punctuation.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from dragonfly import Choice, Repeat
+pfrom dragonfly import Choice, Repeat
```
Comment (LexiconCode) [inline castervoice/lib/ccr/core/punctuation.py]: A typo

**Comment by LexiconCode** [comment]
Deeds Documentation

File: castervoice/lib/ccr/core/nav.py [hunk 2]
```diff
@@ -216,8 +216,11 @@ class Navigation(MergeRule):
         'save':
             R(Key("c-s"), rspec="save", rdescript="Core: Save"),
         'shock [<nnavi50>]':
-            R(Key("enter"), rspec="shock", rdescript="Core: Enter")* Repeat(extra="nnavi50"),
-
+            R(Key("enter"), rspec="shock", rdescript="Enter")* Repeat(extra="nnavi50"),
+        'hard shock [<nnavi50>]':
+            R(Key("s-enter"), rspec="hard shock", rdescript="Hard Shock")* Repeat(extra="nnavi50"),
+        'hard deli [<nnavi50>]':
+            R(Key("s-del"), rspec="hard deli", rdescript="Hard Deli")* Repeat(extra="nnavi50"),
```
Comment (kendonB) [inline castervoice/lib/ccr/core/nav.py]: Shouldn't these be "shin shock" and "shin deli"?

Comment (kendonB) [reply castervoice/lib/ccr/core/nav.py]: `s-del` does nothing in many apps on my system and acts like backspace in RStudio. @alexboche, where do you use it?

**Comment by kendonB** [comment]
@LexiconCode, am I right that [caster quick reference](https://github.com/dictation-toolbox/Caster/blob/develop/castervoice/doc/CasterQuickReference.tex) is currently the only place where stuff like this is documented?

File: castervoice/lib/ccr/core/nav.py [hunk 3]
```diff
@@ -217,7 +217,10 @@ class Navigation(MergeRule):
             R(Key("c-s"), rspec="save", rdescript="Save"),
         'shock [<nnavi50>]':
             R(Key("enter"), rspec="shock", rdescript="Enter")* Repeat(extra="nnavi50"),
-
+        'hard enter [<nnavi50>]':
+            R(Key("s-enter"), rspec="hard enter", rdescript="hard enter")* Repeat(extra="nnavi50"),
+        'hard delete [<nnavi50>]':
```
[the remark this answers is not in the document]
Comment (kendonB) [reply castervoice/lib/ccr/core/nav.py]: shin deli?

File: castervoice/lib/ccr/core/nav.py [hunk 4]
```diff
@@ -217,7 +217,10 @@ class Navigation(MergeRule):
             R(Key("c-s"), rspec="save", rdescript="Save"),
         'shock [<nnavi50>]':
             R(Key("enter"), rspec="shock", rdescript="Enter")* Repeat(extra="nnavi50"),
-
+        'hard enter [<nnavi50>]':
```
[the remark this answers is not in the document]
Comment (kendonB) [reply castervoice/lib/ccr/core/nav.py]: shin deli

**Comment by kendonB** [comment]
"back | shin tabby" will be added in #548. Feel free to make another PR if you really want the other two commands in caster.

**Pull request closed without merging**